routes/web.php

```diff
@@ -14,5 +16,9 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('top');
 });
+Route::get('/', function () {
+    $変数名 = User::all();
+    return view('top', ['変数名' => $変数名 ]);
```

Comment:
こちらも同じく変数名を決めてあげる必要があります。
左の方は画面側で使う変数名、右側はどの変数を渡したいかを定義します。
今回であれば、このように書くと良さそうです。
```suggestion
    return view('top', ['users' => $users ]);
```